addOns/dns/src/main/resources/org/zaproxy/addon/dns/resources/Messages_es_ES.properties

```diff
@@ -0,0 +1,5 @@
+dns.desc = Un Complemento de DNS
```

Comment:
This will be lost, translations are done through [Crowdin](https://crowdin.com/project/zaproxy).